pyqtgraph/parametertree/parameterTypes/numeric.py

```diff
@@ -27,7 +27,7 @@ def makeWidget(self):
         for k in defs:
             if k in opts:
                 defs[k] = opts[k]
-        if 'limits' in opts:
+        if opts.get('limits') is not None:
```

Comment:
is this change meant to address the case where a `limits` key is preset in `opts` but the value of that entry could be `None`?

Reply:
Yeah, in my playing with the new feature, i was auto-generating a bunch of param specs, and setting `'limits'` to `None` blew up. I guess I should make that an actual test, shouldn't I.